spec/controllers/concerns/third_party_transaction_logging_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ThirdPartyTransactionLogging do
+  let(:test_controller_class) do
+    class TestController < ApplicationController
+      extend ThirdPartyTransactionLogging::MethodWrapper
```

Comment:
I disagree with this lint, since this is a spec.  Instantiation a class as a variable here will ensure that it does not pollute the global namespace.